exporter/otlpexporter/config.go

```diff
@@ -19,4 +21,11 @@ type Config struct {
 	configgrpc.ClientConfig `mapstructure:",squash"` // squash ensures fields are correctly decoded in embedded struct.
 }
 
+func (c *Config) Validate() error {
+	if c.SanitizedEndpoint() == "" {
```

Comment:
Why sanitize here? The `newExporter` was not using this func.

Reply:
Yes, I believe this was in effect a miss. The old code checks for "", but fails to check for "http://" or "https://", and since this is dropped afterwards, may be sending an empty string to the gRPC lib.